packages/amazonq/scripts/build/copyFiles.ts

```diff
@@ -78,11 +77,7 @@ async function copy(task: CopyTask): Promise<void> {
     const dst = path.resolve(outRoot, task.destination ?? task.target)
 
     try {
-        await fs.copy(src, dst, {
-            recursive: true,
-            overwrite: true,
-            errorOnExist: false,
-        })
+        await fs.copy(src, dst)
```

Comment:
usually scripts shouldn't call into our app code that depends on vscode's API, since scripts themselves are run by plain node (not vscode-node).

Since node 16 there is `fs.cp` : https://stackoverflow.com/a/69807672/152142